boards/shared/Test_Utils/Test_BaseStateMachineTest.h

```diff
@@ -65,6 +62,9 @@ class BaseStateMachineTest : public testing::Test
                 App_SharedStateMachine_Tick100Hz(state_machine);
             }
 
+            UpdateClock(state_machine, current_time_ms);
+            UpdateSignals(state_machine, current_time_ms);
```

Comment:
I think we should be updating signals after the state machine ticks?

Reply:
@thekenu would be the person to ask about this, since he wrote it